pyuvdata/VERSION

```diff
@@ -1 +0,0 @@
-1.4.2
```

Comment:
Why do we delete this file again? Isn't this where we define our current version?

Reply:
Current version is gotten in `__init__` using magic.

Reply:
MAGIC!! On a serious note though, how do we describe version numbers with scm, I've never used it before. I see where `__version__` is defined in the `__init__` though but not sure where it pulls the context.

Reply:
I don't actually understand the magic in the `__init__` very well. But basically you issue a normal git tag to change the version, and that's it. SCM will automatically get the version based on the git tag -- adding relevant info like whether the repo is dirty and how many commits there have been since the last tag, and what the current hash is if so. When creating a distribution for upload to pypi etc., it will make sure that it is a clean version, and save that information in the upload, so that if a user installs from pypi, they also get the version (even though they don't have the repo).

Reply:
very cool. Thank you.